bleak/backends/scanner.py

```diff
@@ -41,3 +44,18 @@ async def set_scanning_filter(self, **kwargs):
     @abc.abstractmethod
     async def get_discovered_devices(self) -> List[BLEDevice]:
         raise NotImplementedError()
+
+    @classmethod
+    @abc.abstractmethod
+    async def find_specific_device(cls, device_identifier: str, timeout: float = 10.0) -> BLEDevice:
+        """A convenience method for obtaining a ``BLEDevice`` object specified by MAC address or (macOS) UUID address.
```

Comment:
If you care about being technically correct, it is a Bluetooth Address, not a MAC address. Both use the same EUI-48 format but they are not the same thing. But most people will still call a a MAC address anyway. :smile:

Reply:
I do. This is good. Made change.